service/discovery/azure/storage.go

```diff
@@ -218,15 +218,16 @@ func (d *azureStorageDiscovery) handleBlockStorage(disk compute.Disk) (*voc.Bloc
 	}, nil
 }
 
-func handleObjectStorage(account *storage.Account, container storage.ListContainerItem) (*voc.ObjectStorage, error) {
+func handleObjectStorage(account *storage.Account, container storage.ListContainerItem) (*voc.StorageService, error) {
+	//func handleObjectStorage(account *storage.Account, container storage.ListContainerItem) (*voc.ObjectStorage, error) {
 	enc, err := storageAtRestEncryption(account)
 	if err != nil {
 		return nil, fmt.Errorf("could not get object storage properties for the atRestEncryption: %w", err)
 	}
 
-	return &voc.ObjectStorage{
-		Storage: &voc.Storage{
-			CloudResource: &voc.CloudResource{
+	return &voc.StorageService{
```

Comment:
The link to the ObjectStorage is missing. Did we forget something in the ontology? Or am I missing something?

Reply:
@oxisto could you help me with the HttpEndpoint for ObjectStorage/FileStorage/...?